devdata/env-for-consumer.json

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
As far as I know, only `env.json` is automatically used by our tooling. I tried using `rcc` only to create a producer-consumer bot from the template and then run it, but it fails because the bot is expecting environment variable `RC_WORKSPACE_ID`, which tells me that it's not actually using those files and these are hold overs from how we used to simulate the cloud locally within VSCode before our extensions did it.

Reply:
Indeed, but you can run the robot with `rcc run -e path/to/any-env-name.json -t <task-name>` so we still need _devdata/env-for-consumer.json_ and _devdata/env-for-producer.json_ available so we can run the robot outside of VSCode without complaining about any `RC_WORKSPACE_ID`.

Can you try testing that on your side to confirm it is working? (then keep these two files)

----

The _devdata/env.json_ indeed is automatically detected by VSCode, but since the extension can now detect the presence of Work Items JSON files from _devdata_, we can safely remove **ONLY** the _devdata/env.json_ file. (and keep the rest)

Reply:
Okay yup, forgot about that cli flag, I will bring those two files back, thanks.